### ISSUE TITLE
Add easy way to clear a search

### ISSUE THREAD

**Issue by ndufresne** [hotdoc/hotdoc_search_extension#1]
It could be nice if there was a button to clear the search and/or the ability to use the ESC button.

**Comment by MathieuDuponchelle** [comment]
right, FWIW you can use ctrl + backspace as long as the search field has keyboard input, but I agree there should be a nicer way (ESC sounds fine, plus a standard "cross" icon maybe)
